documentation/Forms/FormViewer.stories.mdx

```diff
@@ -56,6 +57,9 @@ There are 2 main parts to the structure:
    `publishingStatus`: If set to false, the link to the form will not appear on the main page, rather under "sandbox"
 
    `displayAlphaBanner`: If set to false, the ALPHA tag will not appear in the header of the page. If not set if will be defaulted to true.
+  
+   `securityAttribute`: It shows an administrator what proceduce should be taken when accessing a form's responses, possible value Protected B, A, Unclassied
```

Comment:
As a piece of documentation, the full string of "Protected A" should be used here, not just "A".